docs/Symfony.md

```diff
@@ -43,10 +43,25 @@ melodiia:
             description: |
                 This is an awesome API. Really.
             version: 1.0.0
+            pagination:
+                max_per_page_attribute: 'FooQueryParameter'
```

Comment:
This is not the default value (you should write the default value here). Which is `max_per_page`.